test/interpreter/function-varp.spec.ts

```diff
@@ -0,0 +1,43 @@
+import {ErrorType, HyperFormula} from '../../src'
+import {adr, detailedError} from '../testUtils'
+import {ErrorMessage} from '../../src/error-message'
+
+describe('Function VAR.P', () => {
+  it('should take at least one argument', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VAR.P()'],
+    ])
+
+    expect(engine.getCellValue(adr('A1'))).toEqualError(detailedError(ErrorType.NA, ErrorMessage.WrongArgNumber))
+  })
+
+  it('should calculate variance (population)', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VAR.P(2, 3)'],
+      ['=VAR.P(1)'],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toEqual(0.25)
+    expect(engine.getCellValue(adr('A2'))).toEqual(0)
+  })
+
+  it('should coerce explicit argument to numbers', () => {
+    const engine = HyperFormula.buildFromArray([
+      ['=VAR.P(2, 3, 4, TRUE(), FALSE(), "1",)'],
+    ])
+    expect(engine.getCellValue(adr('A1'))).toBeCloseTo(1.95918367346939, 6) //inconsistency with product #1
+  })
+
+  it('should ignore non-numeric values in ranges', () => {
```

Comment:
VAR.P has the same description and the same values but different result. I think that the description is not precise enough. One includes logical values the other dosn't.

After "If you want to include logical values and text representations of numbers in a reference as part of the calculation, use the VARPA function." I would go with something like:

```suggestion
  it('should ignore non-numeric values in ranges, including ignoring logical values and text representation of numbers', () => {
```